core/src/main/java/org/mule/runtime/core/transformer/types/DataTypeBuilderFactoryImpl.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.transformer.types;
+
+import org.mule.runtime.api.message.MuleMessage;
+import org.mule.runtime.api.metadata.DataType;
+import org.mule.runtime.api.metadata.DataTypeBuilder;
+import org.mule.runtime.api.metadata.DataTypeBuilderFactory;
+import org.mule.runtime.api.metadata.MimeTypes;
+
+import javax.activation.DataHandler;
+import javax.activation.DataSource;
+
+public class DataTypeBuilderFactoryImpl extends DataTypeBuilderFactory
```

Comment:
Use of `Impl` is against coding guidelines. Instead qualify the name of the factory with something descriptive.